onboarding/geo/README.md

```diff
@@ -0,0 +1,334 @@
+
+## An Introduction to Satellite Bands
+
+To make the most of satellite imagery, it's essential to understand the concept of satellite bands. Satellite bands refer to the specific regions of the electromagnetic spectrum that satellites capture and record. Each band provides unique information about the Earth's surface and atmosphere, making them essential for various geospatial analyses.
+
+### Understanding Satellite Bands
+
+Satellite sensors are equipped with detectors that measure electromagnetic radiation in different parts of the spectrum. Here's an overview of the key satellite bands and their characteristics:
+
+### 1. **Visible Bands:**
+
+   - **Blue Band:** Captures blue light (approximately 0.45-0.52 micrometers). Useful for detecting water bodies and distinguishing water from other land features.
+
+   - **Green Band:** Captures green light (approximately 0.53-0.59 micrometers). Helps identify healthy vegetation, as chlorophyll reflects green light.
+
+   - **Red Band:** Captures red light (approximately 0.63-0.69 micrometers). Useful for vegetation health assessment and land cover classification.
+
+### 2. **Infrared Bands:**
+
+   - **Near-Infrared (NIR) Band:** Captures near-infrared light (approximately 0.75-1.4 micrometers). Used for vegetation analysis, as healthy vegetation strongly reflects NIR light.
+
+   - **Shortwave Infrared (SWIR) Band:** Captures shortwave infrared light (approximately 1.4-3.0 micrometers). Helps detect moisture content in soils and rock types.
+
+### 3. **Thermal Infrared Bands:**
+
+   - **Thermal Infrared (TIR) Band:** Captures thermal radiation (approximately 8-15 micrometers). Measures temperature variations on the Earth's surface, used in applications like detecting wildfires and monitoring urban heat islands.
+
+### 4. **Microwave Bands:**
+
+   - **Microwave Bands:** Microwaves (wavelengths ranging from millimeters to meters) are used in radar imagery. Microwaves can penetrate clouds and provide information on surface roughness, soil moisture, and more.
+
+Understanding these satellite bands and their specific uses is crucial for interpreting satellite imagery. Depending on your geospatial analysis goals, you may need to work with data from multiple bands to extract meaningful insights.
+
+#### Useful Links
+
+- [Introductory Video to Satellite Imagery](https://www.youtube.com/watch?v=ERPgsFSUfX4)
+- [What is Remote Sensing](https://www.youtube.com/watch?v=F2mQ6fJSxRY)
+- [Remote Sensing and Classification](https://www.youtube.com/watch?v=Hgf3k981Cvw)
+- [Landsat8 Example - Band by Band](https://www.youtube.com/watch?v=A6WzAc1FTeA)
+- [Remote Sensing Image Acquisition, Analysis, and Applications - UNSW Specialism](https://www.coursera.org/learn/remote-sensing)
+- [University of Toronto Satellite Imagery in a GIS Specialism](https://www.coursera.org/learn/spatial-analysis-satellite-imagery-in-a-gis)
+
+## Commonly Used NASA and ESA Data Sets
+
+### NASA Data Sets
```

Comment:
It would be awesome to point to where the data can be downloaded to help whenever we do data science / machine learning